web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1374,6 +1374,28 @@ $.extend(fixmystreet.set_up, {
         if (fixmystreet.page !== 'around' && fixmystreet.page !== 'new' && !$('#toggle-fullscreen').length) {
             $('#sub_map_links').append('<a href="#" id="toggle-fullscreen" data-expand-text="'+ translation_strings.expand_map +'" data-compress-text="'+ translation_strings.collapse_map +'" >'+ translation_strings.expand_map +'</span>');
         }
+
+        // Implement triple tap for skipping the map
```

Comment:
I don't understand what this is doing - if they've tapped this link to skip, it can follow it immediately? Why are they tapping three times? There's nothing preventing it working like a normal link, so one tap will follow it. Is this needed?